packages/rancher-cis-benchmark/charts/templates/benchmark-rke2-cis-1.7-permissive.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: cis.cattle.io/v1
+kind: ClusterScanBenchmark
+metadata:
+  name: rke2-cis-1.7-permissive
+spec:
+  clusterProvider: rke2
+  minKubernetesVersion: "1.25.0"
```

@JonCrowther shouldn't we specify the `maxKubernetesVersion` as well ? This said we can keep it this way until we release the next CIS version, so that CIS-1.7 covers up any k8s versions above 1.25 .

```suggestion
  minKubernetesVersion: "1.25.0"
  maxKubernetesVersion: "1.25.x"
```

---

As discussed, the `maxKubernetesVersion` will be added to CIS-1.7, once CIS-1.8 will be released.

---

The precedent is to leave the latest version of the benchmark tests open ended until we add a new version of the benchmarks. That way if a user has k8s 1.26 for example, they're still able to get some use from the scans before we add benchmark 1.8